### PR TITLE
rezooming grid to fit all pages simultaneously

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
Now the overview/map should adapt to fit any grid size, tile aspect ratio, and viewport size.

File: www_src/pages/section-2/section-2.jsx [hunk 1]
```diff
@@ -139,10 +151,39 @@ var Grid = React.createClass({
       layout: newLayout
     });
   },
+  componentDidMount: function () {},
+  componentDidUpdate: function () {
+    var elGrid = React.findDOMNode(this);
```
Comment (k88hudson) [inline www_src/pages/section-2/section-2.jsx]: `this.getDOMNode()`, no?

File: www_src/pages/section-2/section-2.jsx [hunk 2]
```diff
@@ -139,10 +151,39 @@ var Grid = React.createClass({
       layout: newLayout
     });
   },
+  componentDidMount: function () {},
```
Comment (k88hudson) [inline www_src/pages/section-2/section-2.jsx]: empty function

**Comment by k88hudson** [comment]
R+ [[unflagged]](http://plzkthx.herokuapp.com)

**Pull request merged**